### PR TITLE
Add release notes for v23.2.8

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes REL-1116

### CODE REVIEW

**Review by kvoli — APPROVED** [review]

**Review by nvanbenschoten — APPROVED** [review]

**Review by DrewKimball — APPROVED** [review]

File: src/current/_data/releases.yml [hunk 1]
```diff
@@ -6307,3 +6307,31 @@
     docker_arm_limited_access: false
   source: true
   previous_release: v24.2.0-alpha.1
+
+- release_name: v23.2.8
+  lts: true
+  major_version: v23.2
+  release_date: '2024-07-16'
```
Comment (taroface) [inline src/current/_data/releases.yml:6314]: Might need to update this if it goes out today

File: src/current/_includes/releases/v23.2/v23.2.8.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+## v23.2.8
+
+Release Date: July 16, 2024
```
Comment (taroface) [inline src/current/_includes/releases/v23.2/v23.2.8.md:3]: Ditto

**Review by taroface — APPROVED** [review]
LGTM! Just be sure to update the release date if necessary.

File: src/current/_data/releases.yml (hunk 1 above)
Comment (rmloveland) [reply src/current/_data/releases.yml]: Just updated thanks - release went out late last night west coast US time

File: src/current/_includes/releases/v23.2/v23.2.8.md (hunk 2 above)
Comment (rmloveland) [reply src/current/_includes/releases/v23.2/v23.2.8.md]: ✅

**Pull request merged**